linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_login_defs/tests/wrong_multiple.fail.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-sed -i '/UMASK/d' /etc/bashrc
-echo "UMASK 077" >> /etc/bashrc
-echo "UMASK 077" >> /etc/bashrc
-echo "UMASK 077" >> /etc/bashrc
```

Comment:
This file probably should be removed completely because the OVAL check isn't expected to fail when there are multiple values in /etc/login.defs. It fails in the CI.